src/Microsoft.Fast.Components.FluentUI/DesignTokens.cs

```diff
@@ -0,0 +1,189 @@
+﻿using Microsoft.JSInterop;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+public class DesignTokens
+{
+    public const string bodyFont = "bodyFont";
```

Comment:
I wonder if we should pull these out into something like `DesignTokenNames`. It might bet a bit odd in intellisense seeing these members side by side with the token fields.

Reply:
Yeah, that's what I mentioned in the PR text. Maybe make these private. Can't imagine where/when else these would be needed.

Reply:
I was in need of a break I think. Making them private would make it a bit hard for users to use them...
Separate class looks like a good solution. I'll work on that.

Reply:
Done!